pkgs/tools/misc/direnv/default.nix

```diff
@@ -16,15 +15,25 @@ buildGoModule rec {
   # we have no bash at the moment for windows
   BASH_PATH =
     lib.optionalString (!stdenv.hostPlatform.isWindows)
-    "${bash}/bin/bash";
+      "${bash}/bin/bash";
 
   # replace the build phase to use the GNUMakefile instead
   buildPhase = ''
     make BASH_PATH=$BASH_PATH
   '';
 
+  outputs = [ "out" ] ++ map (sh: "${sh}_interactiveShellInit");
```

Comment:
Why do we have once the shells at the end and here at the front? Not very consistent.

Reply:
I don't understand

Reply:
ok now I get it,it was an error